src/Linux/Native/RunAsUser/main.cpp

```diff
@@ -41,6 +40,16 @@ FILE *input = nullptr, *output = nullptr, *error = nullptr;
 #define RTVS_AUTH_BAD_INPUT   201
 #define RTVS_AUTH_NO_INPUT    202
 
+#define RTVS_JSON_MSG_NAME      "name"
```

Comment:
Can you make these all constants rather than defines? `constexpr int` for numbers, `constexpr char[]` for strings.